packages/client-sdk-web/src/errors/cache-service-error-mapper.ts

```diff
@@ -118,16 +106,7 @@ export class CacheServiceErrorMapper
       case StatusCode.RESOURCE_EXHAUSTED:
         return new LimitExceededError(...errParams);
       case StatusCode.ALREADY_EXISTS: {
-        let errCause = '';
-        // TODO: Remove this once the error message is standardized on the server side
-        const errorMessage = errParams[0]?.toString();
-        const isStoreAlreadyExists =
-          errorMessage?.includes('Store with name:') &&
-          errorMessage?.includes('already exists');
-        // If errCause is not already set to 'store_already_exists', check for store_already_exists error
-        if (!errCause && isStoreAlreadyExists) {
-          errCause = 'store_already_exists';
-        }
+        const errCause = errParams[2]?.['err']?.[0];
```

Comment:
Can we confirm from Joseph if he added the change for "store_already_exists" error key too?